pages.zh/common/bmaptool.md

```diff
@@ -1,20 +1,20 @@
 # bmaptool
 
-> 便捷地创建或复制块文件映射（被设计的比`cp`或`dd`更快）.
-> 更多信息: <https://source.tizen.org/documentation/reference/bmaptool>.
+> 便捷地创建或复制块文件映射（被设计的比`cp`或`dd`更快）。
+> 更多信息： <https://source.tizen.org/documentation/reference/bmaptool>.
 
-- 使用图片生成块图文件:
+- 使用图片生成块图文件：
 
-`bmaptool create -o {{blockmap格式文件.bmap}} {{图片文件}}`
+`bmaptool create -o "{{blockmap 格式文件.bmap}}" {{图片文件}}`
```

Comment:
I wonder what's the purpose of using `"`.

Reply:
All else punctuations look good.

Reply:
Interesting, those quotation marks were added by @ bl-ue in 4dc03a386ff2b4509700733a3278ee0b76751ca2, but that was probably not on purpose. I'll remove them again.